src/vs/base/common/htmlContent.ts

```diff
@@ -15,6 +15,11 @@ export interface IMarkdownString {
 	uris?: { [href: string]: UriComponents };
 }
 
+export enum MarkdownStringTextNewlineStyle {
```

Comment:
`const enum` will inline the values, this should generally always be used unless you're exposing an API